trunk/README.txt

```diff
@@ -4,15 +4,15 @@ Donate link: https://www.rto.de
 Tags: elementor, conditions, dynamic, fields, rto
 Requires at least: 4.7
 Tested up to: 5.3
```

Comment:
Maybe update the "Tested up to" value too?

Reply:
Currently there are only Wordpress 5.2 out. 5.3 is in development. Currently there is no 5.4